components/core/src/streaming_archive/writer/Archive.cpp

```diff
@@ -265,13 +265,15 @@ namespace streaming_archive { namespace writer {
 
     void Archive::write_msg (File& file, epochtime_t timestamp, const string& message, size_t num_uncompressed_bytes) {
         vector<encoded_variable_t> encoded_vars;
-        EncodedVariableInterpreter::encode_and_add_to_dictionary(message, *m_logtype_dict_entry_wrapper, m_var_dict, encoded_vars);
+        // ids of variable type in the message
```

Comment:
* Don't think you need the comment.
* How about `var_ids`? Since a reader might think `added_var_ids` are only the new dictionary IDs that were added whereas it's really the dictionary IDs in this message.